test/e2e/performanceprofile/functests/utils/label/label.go

```diff
@@ -0,0 +1,38 @@
+package label
+
+// The label package contains a list of labels that can be used in
+// the functests to indicate and point certain behavior or characteristics
+// of the various tests.
+// Those can be filtered/focused by ginkgo before the test runs.
+
+const (
+	// Slow means test that usually requires reboot or takes a long time to finish.
+	Slow = "slow"
+
+	// ReleaseCritical is for tests that are critical for a successful release of the component that is being tested.
+	ReleaseCritical = "release-critical"
+
+	// SpecializedHardware is for tests that need special Hardware like SR-IOV devices, specific NICs, etc.
+	SpecializedHardware = "specialized-hardware"
+)
+
+// FeatureTag is a label indicates the feature that being tested.
+type FeatureTag string
```

Comment:
I'm not sure about the new type since it requires casting to string when using under `ginkgo.label` which accepts `string`

Reply:
the new type definitions still help somehow because the golang compiler can ensure we don't mix the types - e.g we can only assign one of the defined `FeatureTag`s and not a random string.

Going to be a tradeoff anyway

Reply:
maybe `Kind for those` and `Feature` instead of `FeatureTag`?

Reply:
Sounds good